### ISSUE TITLE
Recommend Scala Metals + VSCode as an IDE on this page: https://www.playframework.com/documentation/2.9.x/IDE

### ISSUE THREAD

**Issue by tanin47** [playframework/playframework#12485]
I've switched from IntelliJ to VSCode + Scala Metals for a while now, and it has been working well except for the twirl template. More importantly it is free.

It seems like it should be recommended here: https://www.playframework.com/documentation/2.9.x/IDE

Thank you

**Comment by mkurz** [comment]
Yes this page needs some love... Feel free to provide a pull request, thanks!
